README.md

```diff
@@ -55,383 +56,251 @@ of the [Federated Identity Community Group](https://fedidcg.github.io/).
 
-Another use example, provided below, shows how to request a credential from one of many IDPs the user may have already linked to this page.
+### `branding` ✔️
 
-```js
-let credential = await navigator.credentials.get({
-  identity : {
-    providers : [
-      {
-        origin: "https://login.idp.net",
-      },
-      // ... many allowed ...
-      {
-        origin: "https://auth.example.biz",
-      },
-    ]
-  }
-});
-```
+Supplying this in `n.l.setStatus` should be fine, with the caveat that the icon URLs provided must be retrieved when `setStatus` is called, *not* when the RP calls `n.c.get`.
 
-### FedCM Integration
+### `login_url` ❗
 
-There are two main points that need to be integrated with FedCM. First is the provider list. The approach we take is to restrict each provider entry to either the `loginURL` member or the `configURL` member. Second is the interaction of this proposal's login with "button mode" FedCM. We allow them to coexist by saying that a FedCM request with `mode: 'button'` implies a `loginTarget: "popup"`. This is for developer convenience.
+The primary caveat is that if a `login_url` is supplied in `setStatus`, it could include identifying information for the user before the user has allowed linking between the RP and IdP. This can be partially alleviated by not supporting an RP-supplied `loginHint` or `domainHint` that would allow the IdP to create that linkage too early in the user journey for user permission to be gathered. The user agent can also choose to require additional confirmation before navigating to the loginUrl.
 
-```js
-navigator.credentials.get({
-  identity: {
-    mode: "button", // loginTarget: "redirect" would cause an error now.
-    providers : [
-      {
-        configURL : "https://example.com/FEDCM.json",
-      },
-      {
-        origin : "https://login.idp.net", // Actually fine!
-      },
-      {
-        loginURL : "https://auth.example.biz/login" // Invalid combination, can't have loginURL and other providers
-      },
-      {
-        configURL : "https://example.com/FEDCM.json", // This provider entry will never be valid,
-        loginURL : "https://auth.example.biz/login"   // even if it is the only one in the list.
-      },
-    ]
-  }
-})
-```
+## Interaction with other FedCM Features
 
-## Relying Party API, Using a Credential
+### "Login-Status" Headers
 
-The RP can use the Credential as an object once it is obtained, as it would with FedCM. This can be used to verify that the user has selected an account with a given IdP, providing an `origin` field on the credential by analogy to the `configUrl` from the [multi IdP proposal](https://github.com/w3c-fedid/multi-idp). It also provides access to a token from the IDP, provided that the `tokenURL` parameter was provided when the credential was stored.
+An IdP sending `Login-Status: logged-in` refreshes the expiration timer on the stored profile information.
```

Comment:
Could you add a little more detail to this? Why do we think that this is the right choice, and how does it refresh it? Just reset the counter?

Reply:
Added some detail about this, as well as more detail in the open questions about how it should interact with the header.